Build failed with MSVC 2008

When I was building mruby with MSVC 2008, the compiler said when it was compiler.c mruby-bin-debugger cmdbreak.c:

> error C2065: “i”: undeclared identifier
> error C2143: syntax error: lack of ";" (before "type")

What is caused by:

```
#define STRTOUL(ul,s) \
    ul = 0; \
    for(int i=0; ISDIGIT(s[i]); i++) ul = 10*ul + (s[i] -'0');
```

It defined an "i" not at the beginning of a scope block.

Another problem is that the compiler didn't find "isblank".

I'm building the master branch, with MSVC 2008, on Windows 7. Here is the original output of "ruby minirake":

```
cmdbreak.c
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(75) : error C4013: “isblank”未定义；假设外部返回 int
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2143: 语法错误 : 缺少“;”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2143: 语法错误 : 缺少“;”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2143: 语法错误 : 缺少“)”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2143: 语法错误 : 缺少“;”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2065: “i”: 未声明的标识符
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2065: “i”: 未声明的标识符
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2065: “i”: 未声明的标识符
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2059: 语法错误 : “)”
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2146: 语法错误 : 缺少“;”(在标识符“l”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(82) : error C2065: “i”: 未声明的标识符
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2143: 语法错误 : 缺少“;”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2143: 语法错误 : 缺少“;”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2143: 语法错误 : 缺少“)”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2143: 语法错误 : 缺少“;”(在“类型”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2065: “i”: 未声明的标识符
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2065: “i”: 未声明的标识符
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2065: “i”: 未声明的标识符
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2059: 语法错误 : “)”
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2146: 语法错误 : 缺少“;”(在标识符“l”的前面)
C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c(271) : error C2065: “i”: 未声明的标识符
(in C:/path/to/mruby-master)
CC    mrbgems/mruby-bin-debugger/tools/mrdb/cmdbreak.c -> build/host-debug/mrbgems/mruby-bin-debugger/tools/mrdb/cmdbreak.obj
rake aborted!
Command Failed: [cl.exe /c /nologo /W3 /we4013 /Zi /MD /O2 /D_CRT_SECURE_NO_WARNINGS /DENABLE_DEBUG /DMRBGEM_MRUBY_BIN_DEBUGGER_VERSION=0.0.0 /I"C:\path\to\mruby-master\include" /Fo"C:\path\to\mruby-master\build\host-debug\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.obj" "C:\path\to\mruby-master\mrbgems\mruby-bin-debugger\tools\mrdb\cmdbreak.c"]
```

--- a/include/mruby.h
+++ b/include/mruby.h
@@ -359,7 +359,7 @@ MRB_API mrb_value mrb_obj_clone(mrb_state *mrb, mrb_value self);
 #define ISALPHA(c) (ISASCII(c) && isalpha((int)(unsigned char)(c)))
 #define ISDIGIT(c) (ISASCII(c) && isdigit((int)(unsigned char)(c)))
 #define ISXDIGIT(c) (ISASCII(c) && isxdigit((int)(unsigned char)(c)))
-#define ISBLANK(c) (ISASCII(c) && isblank((int)(unsigned char)(c)))
+#define ISBLANK(c) (ISASCII(c) && ((c) == ' ' || (c) == '\t'))
 #define ISCNTRL(c) (ISASCII(c) && iscntrl((int)(unsigned char)(c)))
 #define TOUPPER(c) (ISASCII(c) ? toupper((int)(unsigned char)(c)) : (c))
 #define TOLOWER(c) (ISASCII(c) ? tolower((int)(unsigned char)(c)) : (c))

